presto-client/src/client.ts

```diff
@@ -46,7 +63,19 @@ export class PrestoClient {
       this.headers['X-Presto-Time-Zone'] = this.timezone
     }
 
-    // TODO: Set up auth
+    if (authorizationToken) {
+      this.headers['Authorization'] = authorizationToken
+    } else if (basicAuthentication) {
+      // Note this is only available for Node.js
+      this.headers['Authorization'] = `Bearer ${Buffer.from(
```

Comment:
I believe for basic auth, the auth scheme is `Basic` and `Bearer` is for auth token

Reply:
You're right, I fixed it now

Reply:
So the question now is for the auth token, Do you want users to pass in `Bearer <token>` or just `<token>`?
For the latter case, you need to add `Bearer` as the auth scheme in line 67

Reply:
True. What do you think makes more sense?

Reply:
I vote for adding the `Bearer` auth scheme in the `client.ts` and users only pass in the auth token

Reply:
@yhwang I just sent a commit with the change you mentioned 👍